src/coreclr/vm/threads.cpp

```diff
@@ -8198,9 +8198,11 @@ void Thread::StaticInitialize()
 #ifdef FEATURE_SPECIAL_USER_MODE_APC
     InitializeSpecialUserModeApc();
 
+#if defined(TARGET_AMD64)
     // When CET shadow stacks are enabled, support for special user-mode APCs with the necessary functionality is required
     _ASSERTE_ALL_BUILDS(!AreCetShadowStacksEnabled() || UseSpecialUserModeApc());
```

Comment:
We should enable `AreCetShadowStacksEnabled` on Arm64 instead and make it return false for now. The shadow stacks are coming to Arm64 too - https://lwn.net/Articles/940403/.

Reply:
Interesting, I assumed that pointer authentication was taking that space on arm64

I will make `AreCetShadowStacksEnabled` available on arm64 as well.

Reply:
Shadow stacks are stronger mitigation (defense in depth) than pointer authentication of return addresses.

Reply:
Enabled AreCetShadowStacksEnabled on arm64 with trivial implementation.